Source/JavaScriptCore/offlineasm/arm64.rb

```diff
@@ -1364,14 +1364,31 @@ def lowerARM64
             # the labels required for the .loh directive.
             $asm.putStr("#if OS(DARWIN)")
             $asm.puts "Ljsc_llint_loh_adrp_#{uid}:"
+
+            $asm.putStr("#if CPU(ARM64_32)")
```

Comment:
I think you could just do `#if CPU(ADDRESS32)`

Reply:
Fixed.